doc/releases/release-notes-3.2.rst

```diff
@@ -90,6 +90,9 @@ Deprecated in this release
    valid for specific bindings to specify like :dtcompatible:`gpio-leds` and
    :dtcompatible:`fixed-partitions`.
 
+* All the Bluetooth mesh Configuration Client API with prefix "bt_mesh_cfg_*"
+  is deprecated in favor of a new API with prefix "bt_mesh_cfg_cli_*".
```

Comment:
```suggestion
* Bluetooth mesh Configuration Client API prefixed with "bt_mesh_cfg_"
  is deprecated in favor of a new API with prefix "bt_mesh_cfg_cli_".
```

Reply:
Looks like the documentation build is failing after removing `*`.

Reply:
What if replace "bt_mesh_cfg_" with \`\`bt_mesh_cfg_\`\`?

Reply:
Yes, please try that :)